pkg/config/config.go

```diff
@@ -9,8 +11,6 @@ import (
 // These constants MUST be equal to the corresponding names in service definition in definitions/install.yaml
 
 const (
-	//KubePolicyNamespace default kyverno namespace
-	KubePolicyNamespace = "kyverno"
```

Comment:
should we set this from the environment, so other areas of the code can reference?

Reply:
Yes, I am setting this variable from the environment.

```
var KubePolicyNamespace = getKubePolicyNameSpace()
func getKubePolicyNameSpace() string {
 	KubePolicyNamespace := os.Getenv("KYVERNO_NAMESPACE")
 	if KubePolicyNamespace == "" {
 		KubePolicyNamespace = "kyverno"
 	}
 	return KubePolicyNamespace
 }
```

Reply:
Pooja, I mean set the variable statically in the "init" block so its available before other code runs

Reply:
I have read and experimented regarding the execution order of a package. Its variable initialisation first and then init function is called. I think leaving it in `Var` should be fine